shell/src/main/java/alluxio/cli/fsadmin/report/CapacityCommand.java

```diff
@@ -232,6 +295,55 @@ private void print(String text) {
     mPrintStream.println(indent + text);
   }
 
+  /**
+   * Initialize values.
+   */
+  private void initValues() {
+    mIndentationLevel = 0;
+    mSumCapacityBytes = 0;
+    mSumUsedBytes = 0;
+    mSumCapacityBytesOnTierMap = new TreeMap<>((a, b) -> (compare(a, b)));
+    mSumUsedBytesOnTierMap = new TreeMap<>((a, b) -> (compare(a, b)));
+    // Use to know how many tier we have
+    mCapacityTierInfoMap = new TreeMap<>((a, b) -> (compare(a, b)));
+    mUsedTierInfoMap = new TreeMap<>((a, b) -> (compare(a, b)));
+  }
+
+  /**
+   * Compares two tier name according to their rank value.
+   *
+   * @param a one of the String to compare
+   * @param b one of the String to compare
+   * @return compared value to sort tree map
+   */
+  private int compare(String a, String b) {
```

Comment:
I would make this a static method rather than an instance method. It does not require knowing the state of `CapacityCommand`

Reply:
changed to static and put in the FileSystemAdminShellUtils
Thanks